internal/ingest/ingest_test.go

```diff
@@ -744,28 +752,27 @@ func TestSkipEarlierAdsIfAlreadyProcessedLaterAd(t *testing.T) {
 }
 
 func TestRecursionDepthLimitsEntriesSync(t *testing.T) {
+	const entriesDepth = 10
+	totalChunkCount := entriesDepth * 2
+
 	srcStore := dssync.MutexWrap(datastore.NewMapDatastore())
 	h := mkTestHost()
 	pubHost := mkTestHost()
-	ing, core, _ := mkIngest(t, h)
+	ing, core, _ := mkIngestWithConfig(t, h, config.Ingest{
+		AdvertisementDepthLimit: 100,
+		EntriesDepthLimit:       10,
```

Comment:
Should this be set using the const `entriesDepth` or is it just a coincidence these have the same value?